models/video-model.js

```diff
@@ -0,0 +1,42 @@
+const mongoose=require("mongoose");
+const {Schema}=mongoose;
+const videoList=require("./video-data");
+
+const videoSchema=new Schema({
+  videoId:{
+    type:String,
+    required:"Video Id is a required attribute",
+    unique:true
+  },
+  thumbnailURL:{
+    type:String,
+    required:"Thumbnail URL is a required attribute"
+  },
+  title:{
+   type:String,
+   required:"title is required"
+  },
+  description:String,
+  channelName:String,
+  published_date:String
```

Comment:
`createdAt` attribute can be used to find out when this video was created but if this is referring to some other date I think it would be more appropriate to declare it with `type:Date`